common/djangoapps/student/models.py

```diff
@@ -56,6 +56,8 @@
 from util.query import use_read_replica_if_available
 from util.milestones_helpers import is_entrance_exams_enabled
 
+from lms.djangoapps.ccx.models import CustomCourseForEdX
```

Comment:
Can you move this within the previous block of imports, after the `import lms.lib.comment...` import?